pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -229,6 +235,10 @@ type Connection struct {
 	// DefaultPermissions if the filesystem needs to check permissions based on the file mode.
 	// Negotiated in INIT.
 	DefaultPermissions bool
+
+	// NoOpen if FUSE implementation doesn't support open operation
+	// Set during FUSE Open
```

Comment:
* Append `.` to the end
* Would it be better to clarify that this is for optimization purpose and its correctness does not matter?

Reply:
Can you elaborate on your criteria of for optimization purpose, I'm not sure why the correctness doesn't matter

Reply:
discussed offline